migrations/sqls/20190310135013-inital-ddl-up.sql

```diff
@@ -0,0 +1,96 @@
+/* Replace with your SQL commands */
+/*dev_trap DDL */
+
+/*TraPO - Data defintion language */
+
+--USE dev_trapo;
+
+/*Create person.person table */
+CREATE TABLE IF NOT EXISTS person (
+  personid int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  persontype nvarchar(32) NOT NULL, /*PersonType contraint at front end */
+  firstname nvarchar(255) NOT NULL,
```

Comment:
Just a question -- I'm curious why you chose all lowercase for the field names, even when there are multiple words. A couple alternatives would be "snake case" (ie `first_name`) or camelCase/capital case (`firstName` or `FirstName`). The most common that I've seen for MySQL databases is to use snake case and insert an underscore between works. See https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html for more information

This is in the area of subjective preference and I don't have super strong feelings about it myself.